obsidianirc.c

```diff
@@ -0,0 +1,640 @@
+/* Copyright © 2025 Valware & ObsidianIRC Team
+ * License: GPLv3
+ * Name: third/obsidian-register
+ */
+/*** <<<MODULE MANAGER START>>>
+module
+{
+        documentation "https://github.com/link";
```

Comment:
```suggestion
        documentation "https://github.com/ObsidianIRC/UnrealIRCd-Modules";
```

Idk maybe we link the wiki in the other repo